app/models.py

```diff
@@ -30,6 +32,8 @@ class Service(db.Model):
                                nullable=False)
     data = db.Column(JSON)
 
+    status = db.Column(db.String, index=False, unique=False, nullable=False)
```

Comment:
That is a fair point. To the question of whether status should be indexed. According to some random on the internet low cardinality columns shouldn't be indexed at all.
http://image.slidesharecdn.com/fivestepsperform2009-101006154147-phpapp01/95/5-steps-to-postgresql-performance-38-728.jpg?cb=1286398019